### PR TITLE
[ 🇩🇪✨] add horses CP

### PR DESCRIPTION
**Pull request by Cr4zyc4k3**
**What kind of update does this PR provide?** *Please check*

- [ ] new translations / updated translations / translation fixes
- [x] a new feature
- [ ] a new module
- [ ] a bugfix for an existing feature / module
- [ ] improvement of style or user experience
- [ ] other: *Please fill out*

**What is included in your update?**
* Horses
* updated translations for abc
* fixed bug foo
* ...

**Additional notes**
Please add any further notes here!

### CODE REVIEW

File: src/i18n/de_DE.ts
Comment (jxn-30) [inline src/i18n/de_DE.ts:1]: Add Reiterstaffel to `buildingCategories`? (Line 75)

File: src/i18n/de_DE/buildings.ts [hunk 1]
```diff
@@ -1669,4 +1685,45 @@ export default {
...
+        levelcost: ['Keine Ausbaustufen möglich'],
+        maxBuildings: 'Keine Grenze',
+        maxLevel: 0,
+        special: '',
+        startPersonnel: 8,
+        startParkingLots: 6,
+        startVehicles: [],
+        schoolingTypes: ['Polizei'],
+        schools: [8],
+        icon: 'house-medical',
```
Comment (jxn-30) [inline src/i18n/de_DE/buildings.ts:1727]: don't we want smth with a horse or horses as icon?

**Review by jxn-30 — CHANGES_REQUESTED** [review]

File: src/i18n/de_DE/buildings.ts (hunk 1 above)
Comment (Cr4zyc4k3) [reply src/i18n/de_DE/buildings.ts:1727]: https://fontawesome.com/icons/horse ? There is no good icon for that :smile:

Comment (jxn-30) [reply src/i18n/de_DE/buildings.ts]: yeah, I think that is an okayish icon :)

**Review by jxn-30 — APPROVED** [review]

**Pull request merged**